Sources/MapboxCoreNavigation/PassiveNavigationFeedbackType.swift

```diff
@@ -0,0 +1,105 @@
+import Foundation
+
+/**
+ Feedback type is used to specify the type of feedback being recorded with `NavigationEventsManager.sendPassiveNavigationFeedback(_:type:description:)`.
+ */
+public enum PassiveNavigationFeedbackType: CustomStringConvertible {
+    
+    /// Indicates an incorrect visual.
+    case incorrectVisual(subtype: PassiveNavigationIncorrectVisualSubtype?)
+    
+    /// Indicates a road closure was observed.
+    case roadIssue(subtype: PassiveNavigationRoadIssueSubtype?)
+    
+    /// Indicates a wrong traffic.
+    case wrongTraffic(subtype: PassiveNavigationWrongTrafficSubtype?)
+    
+    /// The user’s location is not matching the real position of the vehicle.
+    case badGPS
+    
+    /// Indicates a custom feedback type and subtype.
+    case custom(type: String, subtype: String?)
+    
+    /// Indicates other feedback. You should provide a `description` string to `NavigationEventsManager.sendPassiveNavigationFeedback(_:type:description:)`
+    /// to elaborate on the feedback if possible.
+    case other
+    
+    public var description: String {
```

Comment:
Do we really need `CustomStringConvertible`? As I would rather call these properties something like `typeKey` and `subtypeKey`. We can make an extension for the `description` property to concatenate those two.

Reply:
Agree, renamed fields to typeKey and subtypeKey